src/newtondynamics_physics/SysNewton.cpp

```diff
@@ -0,0 +1,428 @@
+/**
```

Comment:
Is this PR rebased properly? I thought this file was added in the previous PR?

Reply:
Nope, this isn't properly rebased yet

Reply:
Now it is